### PR TITLE
feat: Updated RxJS to ^6.3.0

### PR DESCRIPTION
**Pull request by benlesh**
Updates RxJS to latest minor, 6.3.0 as a requirement

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6011,6 +6011,12 @@ rxjs@^6.0.0:
   dependencies:
     tslib "^1.9.0"
 
+rxjs@^6.3.0:
```
Comment (IgorMinar) [inline yarn.lock]: I'd expect that we'd also drop the current version. Does anyone still depend on it?

**Review by IgorMinar — CHANGES_REQUESTED** [review]
you'll need to update one or more yarn.lock files as well - see CI failures.

File: integration/bazel/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "dependencies": {
     "reflect-metadata": "file:../../node_modules/reflect-metadata",
-    "rxjs": "file:../../node_modules/rxjs",
+    "rxjs": "^6.3.0",
```
Comment (gkalpak) [inline integration/bazel/package.json]: :thinking: Was this change intentional?

File: integration/hello_world__systemjs_umd/package.json [hunk 3]
```diff
@@ -12,13 +12,13 @@
   },
   "dependencies": {
     "@angular/common": "file:../../dist/packages-dist/common",
-    "@angular/core": "file:../../dist/packages-dist/core",
     "@angular/compiler": "file:../../dist/packages-dist/compiler",
+    "@angular/core": "file:../../dist/packages-dist/core",
```
Comment (gkalpak) [inline integration/hello_world__systemjs_umd/package.json]: :100:

File: integration/hello_world__systemjs_umd/package.json [hunk 4]
```diff
@@ -12,13 +12,13 @@
...
     "@angular/platform-browser": "file:../../dist/packages-dist/platform-browser",
     "@angular/platform-browser-dynamic": "file:../../dist/packages-dist/platform-browser-dynamic",
     "core-js": "file:../../node_modules/core-js",
     "plugin-typescript": "8.0.0",
-    "rxjs": "file:../../node_modules/rxjs",
+    "rxjs": "^6.3.0",
```
Comment (gkalpak) [inline integration/hello_world__systemjs_umd/package.json]: :thinking: Was this change intentional?

**Comment by trotyl** [comment]
Should update `peerDependencies` in `/packages/core/package.json` as well, per https://github.com/angular/angular/issues/25225.

File: integration/hello_world__systemjs_umd/package.json (hunk 4 above)
Comment (IgorMinar) [reply integration/hello_world__systemjs_umd/package.json]: no. this was not supposed to have changed.

Comment (benlesh) [reply integration/hello_world__systemjs_umd/package.json]: Thanks, yeah, I ran `yarn upgrade` in here. Reverted.

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**